models/schema_site_bd.go

```diff
@@ -6,11 +6,12 @@ type SiteBd struct {
 	Value map[string]interface{} `json:",omitempty"`
 }
 
-func NewSchemaSiteBd(ops, path string, anpRef map[string]interface{}, host bool) *SiteBd {
+func NewSchemaSiteBd(ops, path, mac string, anpRef map[string]interface{}, host bool) *SiteBd {
 	var externalepgMap map[string]interface{}
 	externalepgMap = map[string]interface{}{
```

Comment:
We can rename the variable to correct naming and initialize directly with attributes

```suggestion
	siteBdMap := map[string]interface{}{
```

Reply:
make sure to also adjust the return value

Reply:
ok